### DISCUSSION TITLE
Tour Chart average values

### DISCUSSION THREAD

**Discussion by jonfre** [mytourbook/mytourbook#980] [Q&A] [answered]
Tour Chart shows average value lines:
![mtb_chart](https://user-images.githubusercontent.com/1889536/211169787-3125a30e-129a-4ce1-b014-e41e1084c615.png)
How can I remove these? I want to only see the graph behind.

**Comment by wolfgang-ch** [comment, marked as answer]
You have activated the segments (average values) in the tour segmenter

![image](https://user-images.githubusercontent.com/1283445/211209641-5aeab2ea-1c61-4407-b57f-18e460e1a853.png)


https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tour-segmenter#segments-in-tour-chart
